ui/app/rif/actions/index.js

```diff
@@ -892,7 +892,7 @@ function getLuminoNetworkData (tokenAddress) {
         userChannels: 0,
       }
       dispatch(this.getTokens()).then(tokens => {
-        const data = tokens.find(n => n.address === tokenAddress);
+        const data = tokens.find(n => n.address.toLowerCase() === tokenAddress.toLowerCase());
```

Comment:
I don't know if you need to do this, seems to me the addresses should be all checksumed, anyway it doesn't make any harm to check on lowecase i guess.

Reply:
I dunno what happened on this case, but on one of the flows it worked and in the other didn't.

So for being sure I lowercased it,